src/internet_identity/tests/README.md

```diff
@@ -4,11 +4,17 @@ This is a test suite for the Internet Identity canister. To run the tests:
 
 ``` bash
 # Make sure II is built with the "test" flavor
-II_DUMMY_CAPTHA=1 ./scripts/build
+II_FETCH_ROOT_KEY=1 II_DUMMY_CAPTCHA=1 ./scripts/build
```

Comment:
out of curiosity, why do these tests need II_FETCH_ROOT_KEY?

Reply:
Actually, they don't. I added the flag to be consistent with the description in the main README (and the comment here).

But I can also revert that, but then I would not call it the `test` flavor here.

Reply:
just curious :) thanks!